config/config.go

```diff
@@ -8,7 +8,9 @@ import (
 	"io/ioutil"
 	"os"
 	"path"
+	"path/filepath"
 	"regexp"
+	"strings"
```

Comment:
This PR now contains changes that should not be part of it. I guess something went wrong during rebasing?

Reply:
Yes, I must have messed it up. Will give it a look again - sorry about the mess @felixfontein